draft-ietf-gnap-core-protocol.md

```diff
@@ -3186,9 +3187,8 @@ key proof together.
 - When an access token is used with no key proof, this is a bearer token request. This type of
     request is used only for calls to the RS, and only with access tokens that are
     not bound to any key as described in {{response-token-single}}.
-- When neither an access token nor key proof are used, this is an unsecured request. This 
-    type of request is used only for calls to the RS during a discovery phase as
-    described in {{rs-request-without-token}}.
+- When neither an access token nor key proof are used, this is an unsecured request. This type
+    of request is not used in the core protocol of GNAP.
```

Comment:
Ok, but the previous text was clearer as to what we're referring to. Could lonk to the proper section in I-D.draft-ietf-gnap-resource-servers as an example?

Reply:
We could point externally to that, but I took the reference out because one should never be making this kind of unprotected call in any of GNAP's core. So, this bullet was really just supposed to complete the matrix of possibilities and say "don't do this" -- the exception is in the extension that allows bootstrapping.